skietbaan_fe/src/components/RegisterStyles.css

```diff
@@ -29,13 +29,14 @@ body{
   border: 1px solid rgba(0,0,0,0.35);
   padding: 20px;
   background-color: white;
+  position: absolute;
   box-shadow: 2px 2px rgba(0,0,0,0.24);
 }
 @media screen and (min-width:768px)  {
 .centre-login{
   width: 50%;
   left:25%;
-  top:25%
+  top: 30%;
 }
 }
```

Comment:
If you have brackets like structured like this it means your indentation is wrong (I see this is not from your changes but could you please fix it )

Reply:
Alright fixed